src/tool_getparam.c

```diff
@@ -1999,8 +1999,8 @@ static ParameterError opt_bool(struct GlobalConfig *global,
     config->doh_verifystatus = toggle;
     break;
   case C_FALSE_START: /* --false-start */
-    config->falsestart = toggle;
-    break;
+    errorf(config->global, "--false-start is no longer supported");
+    return PARAM_BAD_USE;
```

Comment:
I prefer this gentler way of handling this, as I believe it won't actually hurt any use case when the option is just ignored:

```suggestion
    opt_depr(global, a);
    break;
```